### PR TITLE
Update `golem-messages` to 3.7.0

### PR DESCRIPTION
**Pull request by Jakub89**
Just an update.

### CODE REVIEW

File: concent_api/core/tests/test_unit_validation.py [hunk 1]
```diff
@@ -213,7 +214,7 @@ def setUp(self):
             requestor_ethereum_public_key=encode_hex(REQUESTOR_PUB_ETH_KEY),
             requestor_public_key=encode_hex(REQUESTOR_PUBLIC_KEY),
             want_to_compute_task=WantToComputeTaskFactory(
-                provider_ethereum_public_key=encode_hex(PROVIDER_PUB_ETH_KEY),
+                # provider_ethereum_public_key=encode_hex(PROVIDER_PUB_ETH_KEY),
```
Comment (dybi) [inline concent_api/core/tests/test_unit_validation.py]: please remove

File: concent_api/core/tests/test_unit_validation.py [hunk 2]
```diff
@@ -222,7 +223,7 @@ def setUp(self):
...
```
Comment (dybi) [inline concent_api/core/tests/test_unit_validation.py]: please remove

File: concent_api/core/tests/test_unit_validation.py (hunk 1 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_validation.py]: Done

File: concent_api/core/tests/test_unit_validation.py (hunk 2 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_validation.py]: Done

**Pull request merged**